### DISCUSSION TITLE
Unexpected Line Breaks with Hyperlinked Text

### DISCUSSION THREAD

**Discussion by gitacoco** [GitbookIO/community#799] [0. General bugs and questions]
Hi there, I’d like to understand why embedding hyperlinks within certain parts of the text causes line breaks (as shown in points 1 and 2 in my screenshot), while in some other cases it doesn't (point 3). Point 3 is what I expected and more preferable. Does GitBook support inline hyperlinks without adding a line break? Personally speaking, adding one type of formatting shouldn’t alter another. If this behavior is intentional, could you explain the reasoning behind it?

<img width="732" alt="image" src="https://github.com/user-attachments/assets/fb1c4ecf-293c-4bc3-9744-85b9f47d4f1a">

**Comment by aikoii** [comment]
🧹 Closing this as duplicate of #800
